server/husky.js

```diff
@@ -132,29 +144,30 @@ class Husky {
     Content.type = name
     this.contentTypes.set(name, Content)
   }
-  
+
   setupJobs() {
     let interval = parseInt(process.env.POLL_INTERVAL, 10)
     if (Number.isNaN(interval)) interval = 5000
-    
+
     setInterval(async () => {
       for (let listId of this.requestedLists) {
         await this.fetchAndCacheList(listId)
       }
     }, interval)
   }
-  
+
   async fetchAndCacheList(listId) {
     const params = {
       fields:
-        'desc,descData,labels,name,pos,url,idAttachmentCover,dateLastActivity',
+        'desc,descData,labels,name,pos,url,idAttachmentCover,dateLastActivity,',
       attachments: true,
-      members: true
+      members: true,
+      customFieldItems: true
```

Comment:
What happens if you pass this and custom-fields aren't enabled?

Reply:
It's fine there is a check in the code.